client/a8c-for-agencies/components/site-configurations-modal/index.tsx

```diff
@@ -205,34 +213,41 @@ export default function SiteConfigurationsModal( {
 							onChange={ toggleAllowClientsToUseSiteHelpCenter }
 							checked={ allowClientsToUseSiteHelpCenter }
 							name="is_fully_managed_agency_site"
-							disabled={ isSubmitting }
+							disabled={ isDevSite }
 						/>
-						<label htmlFor="configure-your-site-modal-form__allow-clients-to-use-help-center-checkbox">
-							{ translate(
-								'Allow clients to use the {{HcLink}}WordPress.com Help Center{{/HcLink}} and {{HfLink}}hosting features.{{/HfLink}}',
-								{
-									components: {
-										HcLink: (
-											<a
-												target="_blank"
-												href={ localizeUrl(
-													'https://wordpress.com/support/help-support-options/#how-to-contact-us'
-												) }
-												rel="noreferrer"
-											/>
-										),
-										HfLink: (
-											<a
-												target="_blank"
-												href={ localizeUrl(
-													'https://developer.wordpress.com/docs/developer-tools/web-server-settings/'
-												) }
-												rel="noreferrer"
-											/>
-										),
-									},
-								}
-							) }
+						<label
+							className={ isDevSite ? 'disabled-label' : '' }
+							htmlFor="configure-your-site-modal-form__allow-clients-to-use-help-center-checkbox"
+						>
+							{ isDevSite
+								? translate(
+										'Allow clients to use the WordPress.com Help Center and hosting features.'
+								  )
+								: translate(
+										'Allow clients to use the {{HcLink}}WordPress.com Help Center{{/HcLink}} and {{HfLink}}hosting features.{{/HfLink}}',
```

Comment:
I am having a little bit of mixed feelings whether we should keep the links for the disabled component as well or not. 🤔 

The label looks cleaner without the links, but then the user might be missing the context of what the checkbox is about.

Maybe we could consider updating the link color.

---

Not related to this PR, but I think for the future task (once we have feedback from design) we could also consider adding a bit of an explanation, e.g.: "You will be able to change this setting once the site is launched."

Reply:
Yeah, I was wondering the same.
Now that you say it, I think we should go with the colored links. I updated the PR. Thank you!

Reply:
I have created a separate task so we can get to it later in more detail: https://github.com/Automattic/dotcom-forge/issues/8544.